Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRendererTest.cs

```diff
@@ -458,9 +458,9 @@ private void CheckDataValueField (XmlNode outerSpan, string value)
 
     private string GetClickScript (bool isRequired, bool isAutoPostbackEnabled)
     {
-      return "BocBooleanValue.SelectNextCheckboxValue ('ResourceKey', $(this).parent().children('a')[0], "
-             + "$(this).parent().children('a').children('img')[0], $(this).parent().children('span')[1], "
-             + "$(this).parent().children('input')[0], " + isRequired.ToString().ToLower() + ", "
+      return "BocBooleanValue.SelectNextCheckboxValue ('ResourceKey', this.parentNode.querySelector(':scope>a'), "
+             + "this.parentNode.querySelector(':scope>a').querySelector(':scope>img'), this.parentNode.querySelectorAll('span')[1], "
```

Comment:
We can convert this into a single CSS selector

```suggestion
             + "this.parentNode.querySelector(':scope>a>img'), this.parentNode.querySelectorAll('span')[1], "
```